posthog/api/test/test_decide.py

```diff
@@ -2991,6 +2991,21 @@ def test_decide_new_capture_activation(self, *args):
             self.assertEqual(response.status_code, 200)
             self.assertFalse("analytics" in response.json())
 
+        with self.settings(NEW_ANALYTICS_CAPTURE_TEAM_IDS={"0", "*"}, NEW_ANALYTICS_CAPTURE_SAMPLING_RATE=1.0):
+            response = self._post_decide(api_version=3)
+            self.assertEqual(response.status_code, 200)
+            self.assertTrue("analytics" in response.json())
+            self.assertEqual(response.json()["analytics"]["endpoint"], "/i/v0/e/")
```

Comment:
nit: This test could be made more "real"/cover more cases if we set a known seed and `NEW_ANALYTICS_CAPTURE_SAMPLING_RATE=0.02` (or any other "real" number besides the max).

Reply:
Agreed, if it was long-lived code I'd add a test that runs N queries and computes stats, but it's not worth it here. Manually tested that turning the rate to 50% reduced traffic, although the long-tail of open sessions will means a slow taper-out.